<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profilemanager</class>
 <widget class="QDialog" name="Profilemanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:red</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Profile&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="profile_left">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>481</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange</string>
   </property>
   <item>
    <property name="text">
     <string>Name: </string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Level:</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XP: </string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XP left for next level-up:</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Games played:</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Games won:</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Winrate:</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="profile_right">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>231</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange
</string>
   </property>
   <item>
    <property name="text">
     <string>name</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>placeholder</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>placeholder</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>placeholder</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>placeholder</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>placeholder</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>placeholder</string>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; vertical-align:super;&quot;&gt;Enter name&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>570</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="win_info_label">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>310</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
